packages/lib-classifier/src/components/Classifier/components/TaskArea/components/Tasks/components/TaskNavButtons/TaskNavButtonsContainer.js

```diff
@@ -4,6 +4,10 @@ import { inject, observer, PropTypes as MobXPropTypes } from 'mobx-react'
 import TaskNavButtons from './TaskNavButtons'
 
 function storeMapper (stores) {
+  const {
+    active: feedback,
```

Comment:
While the PFE implementation used `active`, the term is usually used in the FEM when referring to an instance of a resource, so maybe I want to name this boolean attribute something different?